src/lib.rs

```diff
@@ -116,6 +117,10 @@ impl fmt::Display for StartupError {
                 f,
                 "Our Bitcoin interface is bitcoind but we have no 'bitcoind_config' entry in the configuration."
             ),
+            Self::DbMigrateBitcoinTxs(msg) => write!(
+                f,
+                "Error when migrating Bitcoin transaction from Bitcoin backend to database: {}.", msg
```

Comment:
I removed reference to bitcoind so that message applies to Electrum and other backends when available.

Reply:
There will never be a version of Liana shipped with both Electrum and a database version <= 4. Also why i made the code below where this error may be triggered bitcoind-specific.

(no need to change it either way, it's not a big deal)

Reply:
Ah that's true, although a user could in principle change their existing daemon config file to specify an Electrum connection before starting the new version of Liana.